Luft/Luft.m

```diff
@@ -79,4 +94,42 @@ - (void)traceNotifications:(NSNotification *)notification {
     }
 }
 
+- (BOOL)isViewController:(NSString *)filename {
```

Comment:
Why does this method need to be in here? It's already in `IDESourceCodeEditor+Luft.m`